### PR TITLE
Update status component: new color and shape of indicator

### PR DESCRIPTION
**Pull request by adaszyn**
## Hey, I just made a Pull Request!

As my first contribution, I tried to fix `Status` component by applying changes described in https://github.com/spotify/backstage/issues/724. 
### Screenshots
image
image

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by katz95** [comment]
Yay! Looking better! 

Can we combine the status/label sections? The text label should always be paired with the status dot! 😄 

And when the label is "N/A", the dot can be the same grey color that it is when the label is 'Aborted' or 'Disabled.' 

Thank you!!

**Comment by adaszyn** [comment]
@katz95 Is there a use-case for a status component without a text or do we always want to include a label? Thanks!

**Comment by katz95** [comment]
We recommend always pairing it with a label! 😺

**Review by nikek — APPROVED** [review]

File: packages/core/src/components/Status/Status.tsx [hunk 1]
```diff
@@ -57,43 +71,23 @@ const useStyles = makeStyles<BackstageTheme>((theme) => ({
...
 
 export const StatusError: FC<{}> = (props) => {
   const classes = useStyles(props);
   return (
-    <span
-      className={classNames(classes.status, classes.error)}
-      aria-label="Status error"
-      {...props}
-    />
+    <span className={classNames(classes.status, classes.error)} {...props} />
```
Comment (stefanalund) [inline packages/core/src/components/Status/Status.tsx:87]: Why remove the `aria-label`?

Comment (adaszyn) [reply packages/core/src/components/Status/Status.tsx:87]: Aria attributes on non-semantic elements (span, divs) are often omitted by assistive technology so the message was not communicated to a user. As @katz95 mentioned, `Status` component should be used with text - making it accessible without any extra metadata. Additionally, one should keep in mind that `aria-labels` should be translated when applying internationalization (extra complexity). I'm happy to revert that if it doesn't make sense!

Comment (stefanalund) [reply packages/core/src/components/Status/Status.tsx:87]: Oh, good point.

**Pull request merged**